src/AdvApi32.Desktop/AdvApi32.cs

```diff
@@ -218,7 +263,7 @@ public static partial class AdvApi32
         /// </returns>
         [DllImport(api_ms_win_service_management_l2_1_0, SetLastError = true, CharSet = CharSet.Unicode)]
         [return: MarshalAs(UnmanagedType.Bool)]
-        public static unsafe extern bool ChangeServiceConfig2(SafeServiceHandle hService, ServiceInfoLevel dwInfoLevel, void* lpInfo);
+        public static extern bool ChangeServiceConfig2(SafeServiceHandle hService, ServiceInfoLevel dwInfoLevel, IntPtr lpInfo);
```

Comment:
Agreed. I don't understand why either. This is a breaking change because it removes the `void*` overload, yet it doesn't introduce an `IntPtr` overload because the code generator would have created it from the `void*` one. I suspect this is what drove you to change all the xml doc comments that referenced pointer overloads. Can we revert this and all the comment changes?